Broken on master:
```
error: builder for '/nix/store/shlp9cif62i7f0m8qzf16iapjs0j871a-python3.12-strawberry-graphql-0.254.0.drv' failed with exit code 1;
       last 25 log lines:
       > FAILED tests/experimental/pydantic/test_fields.py::test_types[AnyHttpUrl-str] - AssertionError: assert <class 'pydantic.networks.AnyHttpUrl'> is <class 'str'>
       > FAILED tests/experimental/pydantic/test_fields.py::test_types[HttpUrl-str] - AssertionError: assert <class 'pydantic.networks.HttpUrl'> is <class 'str'>
       > FAILED tests/experimental/pydantic/test_fields.py::test_types[PostgresDsn-str] - AssertionError: assert <class 'pydantic.networks.PostgresDsn'> is <class 's...
       > FAILED tests/experimental/pydantic/test_fields.py::test_types[RedisDsn-str] - AssertionError: assert <class 'pydantic.networks.RedisDsn'> is <class 'str'>
       > = 4 failed, 1776 passed, 80 skipped, 8 xfailed, 4 xpassed, 70 warnings in 39.52s =
       > Task was destroyed but it is pending!
       > task: <Task pending name='Task-2475' coro=<RequestHandler.start() running at /nix/store/iilldmc1dgjrs9mb9ykx4swl8zdsqrqy-python3.12-aiohttp-3.11.9/lib/python3.12/site-packages/aiohttp/web_protocol.py:563> wait_for=<Task pending name='Task-2476' coro=<RequestHandler._handle_request() running at /nix/store/iilldmc1dgjrs9mb9ykx4swl8zdsqrqy-python3.12-aiohttp-3.11.9/lib/python3.12/site-packages/aiohttp/web_protocol.py:480> wait_for=<Future finished result=None> cb=[Task.task_wakeup()]>>
       > Task was destroyed but it is pending!
       > task: <Task pending name='Task-2476' coro=<RequestHandler._handle_request() running at /nix/store/iilldmc1dgjrs9mb9ykx4swl8zdsqrqy-python3.12-aiohttp-3.11.9/lib/python3.12/site-packages/aiohttp/web_protocol.py:480> wait_for=<Future finished result=None> cb=[Task.task_wakeup()]>
       > Task exception was never retrieved
       > future: <Task finished name='Task-2639' coro=<BaseGraphQLWSHandler.handle_async_results() done, defined at /build/source/strawberry/subscriptions/protocols/graphql_ws/handlers.py:154> exception=ClientConnectionResetError('Cannot write to closing transport')>
       > Traceback (most recent call last):
       >   File "/build/source/strawberry/subscriptions/protocols/graphql_ws/handlers.py", line 182, in handle_async_results
       >     await self.send_data_message(result, operation_id)
       >   File "/build/source/strawberry/subscriptions/protocols/graphql_ws/handlers.py", line 216, in send_data_message
       >     await self.send_message(data_message)
       >   File "/build/source/strawberry/subscriptions/protocols/graphql_ws/handlers.py", line 219, in send_message
       >     await self.websocket.send_json(message)
       >   File "/build/source/strawberry/aiohttp/views.py", line 113, in send_json
       >     await self.ws.send_str(self.view.encode_json(message))
       >   File "/nix/store/iilldmc1dgjrs9mb9ykx4swl8zdsqrqy-python3.12-aiohttp-3.11.9/lib/python3.12/site-packages/aiohttp/web_ws.py", line 408, in send_str
       >     await self._writer.send_frame(
       >   File "/nix/store/iilldmc1dgjrs9mb9ykx4swl8zdsqrqy-python3.12-aiohttp-3.11.9/lib/python3.12/site-packages/aiohttp/_websocket/writer.py", line 73, in send_frame
       >     raise ClientConnectionResetError("Cannot write to closing transport")
       > aiohttp.client_exceptions.ClientConnectionResetError: Cannot write to closing transport
       For full logs, run 'nix log /nix/store/shlp9cif62i7f0m8qzf16iapjs0j871a-python3.12-strawberry-graphql-0.254.0.drv'.
error: 1 dependencies of derivation '/nix/store/irg43l2g3yzfvira3jgq56c9nrpm3r62-python3.12-strawberry-django-0.47.1.drv' failed to build
error: 2 dependencies of derivation '/nix/store/fbic1v920dd9jz2zzffg7i5ybcg23285-netbox-4.1.3.drv' failed to build
error: 1 dependencies of derivation '/nix/store/z1qlkvm17vkslmrisvfq049ssjdnp4cf-python3.12-netbox-qrcode-0.0.15.drv' failed to build
error: 1 dependencies of derivation '/nix/store/s7vr2ic0qkffnk75i5w071sk2260lq5v-python3.13-netbox-qrcode-0.0.15.drv' failed to build
error: 2 dependencies of derivation '/nix/store/xzkqkc081ya6jpckqydqjbiw97wik3jw-review-shell.drv' failed to build
```

I've seen this problem with `pkgs-unstable.netbox` yesterday too. This pr is working with nixpkgs-24.11.
Waiting for the python3.12-strawberry-graphql to be fixed...

--- a/pkgs/development/python-modules/netbox-qrcode/default.nix
+++ b/pkgs/development/python-modules/netbox-qrcode/default.nix
@@ -1,0 +1,46 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  wheel,
+  pillow,
+  qrcode,
+  netbox,
+}:
+buildPythonPackage rec {
+  pname = "netbox-qrcode";
+  version = "0.0.15";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "netbox-community";
+    repo = "netbox-qrcode";
+    rev = "v${version}";
+    hash = "sha256-gEBOjmX7dcLjiy0u6raAtAaqZUVYsPz3YhupjRRqVDE=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    qrcode
+    pillow
+  ];
+
+  nativeCheckInputs = [ netbox ];
+
+  preFixup = ''
+    export PYTHONPATH=${netbox}/opt/netbox/netbox:$PYTHONPATH
+  '';
+
+  pythonImportsCheck = [ "netbox_qrcode" ];
+
+  meta = {
+    description = "Netbox plugin for generate QR codes for objects: Rack, Device, Cable.";
+    homepage = "https://github.com/netbox-community/netbox-qrcode";
+    changelog = "https://github.com/netbox-community/netbox-qrcode/releases/tag/${src.rev}";
+    license = lib.licenses.asl20;
+    platforms = lib.platforms.linux;
+    maintainers = with lib.maintainers; [ felbinger ];
+  };
+}

--- a/pkgs/top-level/python-packages.nix
+++ b/pkgs/top-level/python-packages.nix
@@ -9138,6 +9138,8 @@ self: super: with self; {
 
   netbox-documents = callPackage ../development/python-modules/netbox-documents { };
 
+  netbox-qrcode = callPackage ../development/python-modules/netbox-qrcode { };
+
   netbox-reorder-rack = callPackage ../development/python-modules/netbox-reorder-rack { };
 
   netbox-plugin-prometheus-sd = callPackage ../development/python-modules/netbox-plugin-prometheus-sd { };

--- a/pkgs/top-level/python-packages.nix
+++ b/pkgs/top-level/python-packages.nix
@@ -9138,11 +9138,11 @@ self: super: with self; {
 
   netbox-documents = callPackage ../development/python-modules/netbox-documents { };
 
+  netbox-plugin-prometheus-sd = callPackage ../development/python-modules/netbox-plugin-prometheus-sd { };
+
   netbox-qrcode = callPackage ../development/python-modules/netbox-qrcode { };
 
   netbox-reorder-rack = callPackage ../development/python-modules/netbox-reorder-rack { };
-
-  netbox-plugin-prometheus-sd = callPackage ../development/python-modules/netbox-plugin-prometheus-sd { };
 
   netcdf4 = callPackage ../development/python-modules/netcdf4 { };
 

